lib/spark.js

```diff
@@ -23,9 +24,11 @@ export default class Spark {
     })
     await assertOkResponse(res, 'Failed to fetch the current SPARK round')
     this.#activity.onHealthy()
-    const { retrievalTasks, ...round } = await res.json()
+    const { retrievalTasks, maxTaskPerNode, ...round } = await res.json()
```

Comment:
is this a typo? We are mixing singular and plural `task` here

Reply:
https://github.com/filecoin-station/spark-api/pull/172/files has `maxTasksPerNode`. I think we should do some more testing here

Reply:
Yeah, looks like a typo.

Do you have any suggestions on how to test this?

I think we also need to deploy https://github.com/filecoin-station/spark-api/pull/172 before we can test this new part.

Reply:
I think for now we would manually test this